### DISCUSSION TITLE
"Real time" or "sneak peek" live chat

### DISCUSSION THREAD

**Discussion by jordan-brough** [chatwoot/chatwoot#4292] [Feature Requests]
Have there been any thoughts around adding "real time" / "sneak peek" functionality to the website live chat? Where agents can see the customer's message in real time as they type it?

Examples:
- https://www.liveagent.com/features/real-time-typing-view/
- https://www.livechat.com/help/how-to-chat-window/#sneak-peek
- https://www.salesforceben.com/an-introduction-to-salesforce-live-agent/ (search for "sneak peek")

**Comment by sojan-official** [comment]
ref: #https://github.com/chatwoot/chatwoot/issues/1137
